Vignettes/variable_details.Rmd

```diff
@@ -90,44 +88,44 @@ kable(varDetails[c(75:79), c(1:5)])
++ For categorical variables that are not changing variable types (i.e. cat to cat), it is ideal to retain the same values as indicated in each CCHS survey. But for transformed categorical variables that have changed in type (i.e cat to cont), you will have to develop values that make the most sense to your analysis. In `variable_details.csv`, variables that have gone from cat to cont have used midpoints of each category.
 
-8. **numValidCat:** the number of categories for a variable. This only applies to variables in which the **toType** is cat. For continuous variables, `numValidCat = N/A`. Not applicable, missing, and else categories are not included in the category count. Note that this column is not necessary for `RecWTable()`.
+8. **numValidCat:** the number of categories for a variable. This only applies to variables in which the **toType** is cat. For continuous variables, `numValidCat = N/A`. Not applicable, missing, and else categories are not included in the category count. Note that this column is not necessary for `rec_with_table()`.
 
 ```{r, echo=FALSE, warning=FALSE}
-kable(varDetails[c(75:79), c(1:8)])
+kable(variable_details[c(75:79), c(1:8)])
 ```
 
 9. **catLabel:** short form label describing the category of a particular variable.
 
 ```{r, echo=FALSE, warning=FALSE}
-kable(varDetails[c(75:79), c(1:9)])
+kable(variable_details[c(75:79), c(1:9)])
 ```
 
 10. **catLabelLong:** more detailed label describing the category of a particular variable. This label should be identical to what is shown in the CCHS data documentation, unless you are creating derived variables and would like to create your own label for it.
 
 ```{r, echo=FALSE, warning=FALSE}
-kable(varDetails[c(75:79), c(1:10)])
+kable(variable_details[c(75:79), c(1:10)])
 ```
 
 11. **units:** the units of a particular variable. If there are no units for the variable, write `N/A`. Note, the function will not work if there different units between the rows of a variable.
 
 ```{r, echo=FALSE, warning=FALSE}
-kable(varDetails[c(75:79), c(1:11)])
+kable(variable_details[c(75:79), c(1:11)])
 ```
 
 12. **recFrom:** the range of values for a particular category in a variable as indicated in the CCHS. See CCHS data documentation for each survey cycle and use the smallest and large values as your range to capture all values between the survey years. 
 
-The rules for each category of a new variable are a string in `recFrom` and value in `recTo`. These recode pairs are the same syntax as `{sjmisc::rec()` -- for more details see [bllflow::RecWTable()](). Recode pairs are obtained from the RecFrom and RecTo columns
+The rules for each category of a new variable are a string in `recFrom` and value in `recTo`. These recode pairs are the same syntax as `{sjmisc::rec()`. Recode pairs are obtained from the RecFrom and RecTo columns
   *multiple values* that are recoded into a new single value are separated with comma, e.g. `recFrom = "1,2"; recTo = 1`
```

Comment:
there is no support for comma????????????????????

Reply:
Will take out the line that discusses multiple values